mcs/class/System/Test/System.Net.Sockets/SocketTest.cs

```diff
@@ -4319,6 +4319,87 @@ public void SendAsyncFile ()
 			}
 		}
 		
+#if NET_4_5
+
+		// Test case for bug #36192
+		[Test]
+		public void BindToIPV4EndPointUsingDualModelSocket() {
+			using(var s = new Socket(SocketType.Stream, ProtocolType.Tcp)) {
+				try {
+					var host = new IPEndPoint(IPAddress.Loopback, 0);
+					
+					s.Bind(host);
+				} catch(SocketException) {
```

Comment:
Sure makes sense it is covered below with the other tests now, will clean up and remove.